packages/core/admin/admin/src/pages/SettingsPage/components/Tokens/Table/index.js

```diff
@@ -83,7 +83,10 @@ const Table = ({
               <Td>
                 {token.lastUsedAt && (
                   <Typography textColor="neutral800">
-                    <RelativeTime timestamp={new Date(token.lastUsedAt)} />
+                    <RelativeTime
+                      timestamp={new Date(token.lastUsedAt)}
+                      customIntervals={[{ unit: 'hours', threshold: 1, text: 'last hour' }]}
```

Comment:
Will this work with localization?

Reply:
no, I am asking rn how we normally handle it and if we should create translationss for this case.

Reply:
I think we could use icu4x to better deal with localization for example for dates
Still this would mean changes and would be a longer process.

 https://github.com/unicode-org/icu4x/blob/main/docs/tutorials/js.md

Reply:
I wanted to take a look first at react-intl, I suspect there is some sort of utility to get internationalized time interval texts, I am looking into it 👀

Reply:
didnt find anything relevant, we might take a look at icu4x in the future, thnks boegie! For now I added the internationalization for some languages.